spanner/changestreams/src/main/java/com/example/spanner/changestreams/ChangeStreamSample.java

```diff
@@ -0,0 +1,255 @@
+      )
+    );
+  }
+
+  // Execute a change stream query, return and print out the result records.
+  public static List<ChangeStreamRecord> executeChangeStreamQueryAndPrint(
+      DatabaseClient dbClient, String changeStreamName, String startTimestamp,
+      String endTimestamp, String partitionToken) {
+    System.out.println("Executing a change stream query with: "
+        + "start_timestamp => " + startTimestamp
+        + ", end_timestamp => " + endTimestamp
+        + ", partition_token => " + partitionToken
+        + ", heartbeat_milliseconds => 5000"
+        + ", read_options => null");
+
+    final String query =
+        String.format("SELECT * FROM READ_%s ("
+          + "start_timestamp => @startTimestamp,"
+          + "end_timestamp => @endTimestamp,"
+          + "partition_token => @partitionToken,"
+          + "read_options => null,"
+          + "heartbeat_milliseconds => @heartbeatMillis"
+          + ")", changeStreamName);
+
+    final ResultSet resultSet =
+        dbClient
+          .singleUse()
+          .executeQuery(
+            Statement.newBuilder(query)
+              .bind("startTimestamp").to(Timestamp.parseTimestamp(startTimestamp))
+              .bind("endTimestamp").to(Timestamp.parseTimestamp(endTimestamp))
+              .bind("partitionToken").to(partitionToken)
+              .bind("heartbeatMillis").to(5000)
+              .build());
+
+    ChangeStreamRecordMapper changeStreamRecordMapper = new ChangeStreamRecordMapper();
+    List<ChangeStreamRecord> res = new ArrayList<>();
+    while (resultSet.next()) {
+      // Parses result set into change stream result format.
+      final List<ChangeStreamRecord> records =
+          changeStreamRecordMapper.toChangeStreamRecords(resultSet.getCurrentRowAsStruct());
+
+      // Prints out all the query results.
+      for (final ChangeStreamRecord record : records) {
+        if (record instanceof DataChangeRecord) {
+          System.out.println("Received a DataChangeRecord: " + record);
+        } else if (record instanceof HeartbeatRecord) {
+          System.out.println("Received a HeartbeatRecord: " + record);
+        } else if (record instanceof ChildPartitionsRecord) {
+          System.out.println("Received a ChildPartitionsRecord: " + record);
+        } else {
+          // We should never receive reach here.
+          throw new IllegalArgumentException("Unknown record type " + record.getClass());
+        }
+      }
+      res.addAll(records);
+    }
+
+    return res;
+  }
+  // [END change_stream_sample_query_change_stream]
+
+  // [START change_stream_sample_drop_change_stream]
+  public static void dropChangeStream(
+      DatabaseAdminClient dbAdminClient, String instanceId, String databaseId, String tableName,
+      String changeStreamName) throws Exception {
+    System.out.println(
+        String.format("Updating database DDL to drop table %s and change stream %s.",
+          tableName, changeStreamName));
+
+    updateDdl(
```

Comment:
nit: maybe it is easier to inline this function to make the sample self-contained?

Reply:
Done.